src/providers/TeamsProvider.ts

```diff
@@ -130,6 +130,21 @@ export class TeamsProvider extends MsalProvider {
    */
   public static microsoftTeamsLib;
 
+  /**
+   * Execute a deeplink against the Teams lib.
+   *
+   * @static
+   * @param {string} deeplink
+   * @param {(status: boolean, reason?: string) => void} [onComplete]
+   * @memberof TeamsProvider
+   */
+  public static executeDeeplink(deeplink: string, onComplete?: (status: boolean, reason?: string) => void): void {
+    const teams = TeamsProvider.microsoftTeamsLib || microsoftTeams;
+    if (teams) {
+      teams.executeDeeplink(deeplink, onComplete);
```

Comment:
This is throwing two exceptions:
1. it can't find `teams.executeDeeplink`. It should be `teams.executeDeepLink`. The function should be renamed to match as well.
2. The teams sdk might not be initialized and throws an exception, call [`teams.initialize`](https://docs.microsoft.com/en-us/javascript/api/@microsoft/teams-js/microsoftteams?view=msteams-client-js-latest#initialize-------void--string---) first.

Reply:
Ahah! You have a working test... Can you send me steps or whatever account I should use to try this out? I tried setting up a tab app for one of the MeganB accounts, but got denied due to lack of permissions in Teams. I would love to know what steps you took.

Reply:
I ended up using a dummy `personDetails` and it worked for me, even without having to auth:

```html
     <script>
      document.querySelector('mgt-person').personDetails = {
        displayName: 'Bill Gates',
        userPrincipalName: 'a valid userPrincipalName in the tenant'
      };
    </script>
```

Reply:
I'm still having a tough time testing in the Teams context, so this hasn't been tested in that way. Can you try out the latest tweaks I made in your setup? I haven't gotten mine to work inside Teams yet.

Reply:
You're a genius! I tested it out and it works great with the latest updates. Let me know what you think.